exceptions/CustomExceptions.java

```diff
@@ -0,0 +1,29 @@
+package exceptions;
+
+public class CustomExceptions {
```

Comment:
Could you create each exception in a separate class?

Reply:
sure. done